libs/application/templates/children-residence-change/src/fields/ChangeInformation/index.tsx

```diff
@@ -9,9 +8,14 @@ import {
   constructParentAddressString,
 } from '../../lib/utils'
 import { newResidence } from '../../lib/messages'
+import { CRCFieldBaseProps } from '../../types'
```

Comment:
Should we change to alias?

We can make a new PR where we change almost all imports to alias